technical/pivots_points.py

```diff
@@ -32,34 +28,21 @@ def pivots_points(dataframe: pd.DataFrame, timeperiod=30, levels=3) -> pd.DataFr
     :return: dataframe
     """
 
-    data = {}
+    typical_price = (dataframe['high'] + dataframe['low'] + dataframe['close']) / 3
+    data = {"pivot": typical_price}
 
-    low = qtpylib.rolling_mean(
-        series=pd.Series(index=dataframe.index, data=dataframe["low"]), window=timeperiod
-    )
+    for i in range(1, levels + 1):
+        data[f"r{i}"] = 2 * data["pivot"] - dataframe['low']
+        data[f"s{i}"] = 2 * data["pivot"] - dataframe['high']
```

Comment:
I'm not sure this is producing the same results, actually (i'm pretty sure it will not).
We used to look at a rolling price (for all 3 necessary units) - which seems to have been removed completely - which will automaticall also completely change results.

I'm not per se against it - but some explanation about your thoughts (why do it this way instead of the other) will for sure be necessary to accept this.

Reply:
But I have implemented using Numpy function,So it should work

Reply:
err ... no ? 
what i mean is - the prior calculation of `high` and `low` was using a rolling mean - in pandas / numpy terms something around `high = dataframe['high'].rolling(timeperiod).mean()` 

not using qtpylib here is certainly preferred - and as said above, i'm also not per se against not using rolling averages (and instead use the price directly) - but i'd like to understand the reason for that change (if it was intentional ...).